apps/client/src/pages/dashboard/components/sections/BookmarkSection.tsx

```diff
@@ -33,75 +35,103 @@ const BookmarkSection = ({
   onAllViewClick,
   isAllViewExpanded = false,
 }: BookmarkSectionProps) => {
-  // 현재 카테고리의 북마크만 필터링
   const filteredBookmarks =
     activeCategory === UI_TEXT.category.all
       ? bookmarks
-      : bookmarks.filter(() => {
-          // 실제로는 카테고리별 필터링 로직이 필요하지만,
-          // 현재는 mock 데이터이므로 간단히 처리
-          return true;
-        });
+      : activeCategory === '안 읽은 정보'
+        ? bookmarks.filter((bookmark) => !bookmark.isRead)
+        : bookmarks.filter((bookmark) => {
+            const category = categories.find(
+              (cat) => cat.text === activeCategory
+            );
+            return category && bookmark.categoryId === category.id;
+          });
+
```

Comment:
중첩 삼항 연산자는 가독성을 낮추는 방식 중 하나에요! 물론 조건을 파악하기가 어렵지 않다면 사용하는 것도 고려할 수 있지만, 만약 조건 파악이 조금 힘들다면 `if/else`가 오히려 더 좋은 방법이 될 것 같아요! `early return 패턴`도 적극적으로 이용하는 것도 좋을 것 같아요 👍 

```suggestion
const filteredBookmarks = (() => {
  if (activeCategory === UI_TEXT.category.all) {
     return bookmarks;
}
  if (activeCategory === '안 읽은 정보') {
    return bookmarks.filter((bookmark) => !bookmark.isRead);
  }

  const category = categories.find((cat) => cat.text === activeCategory);
  return category
    ? bookmarks.filter((bookmark) => bookmark.categoryId === category.id)
    : [];
})();
```

Reply:
넵! 말씀해주신 부분 공감됩니다 🙌
해당 부분은 조건이 살짝 복잡해질 수 있어서 말씀대로 if/else나 early return 쪽으로 리팩터링해보는 게 좋을 것 같아요. 참고해서 개선해보겠습니다! 감사합니다 😊